plugins/standard_tests/newhelm/tests/bbq.py

```diff
@@ -302,7 +303,7 @@ def make_test_items(self, dependency_helper: DependencyHelper) -> List[TestItem]
                     prompts=[
                         PromptWithContext(
                             prompt=Prompt(text=text, options=sut_options),
-                            context=test_question.context,
+                            context=TypedData.from_instance(test_question.context),
```

Comment:
I'm not certain that this would work, but could we add a field validator for `context` in `PromptWithContext` that does the conversion from the context object to TypedData? So that you can just set 
> `context=test_question.context`

Not sure if this is worth exploring. But it might be nice if we want it to be cleaner. And also if we expect this special context functionality to be something used by other Tests.

Reply:
Do you think we'll be able to control the read as well? I want to make sure that if a user accesses `context` they get back the value they expect.

Reply:
That's true. Might not be intuitive.